test/common/README.md

```diff
@@ -402,11 +403,51 @@ Creates a new `Countdown` instance.
 
 Decrements the `Countdown` counter.
 
-### Coutndown.prototype.remaining
+### Countdown.prototype.remaining
 
 Specifies the remaining number of times `Countdown.prototype.dec()` must be
 called before the callback is invoked.
 
+## DNS Module
+
+The `DNS` module provides a naïve DNS parser/serializer.
+
+### readDomainFromPacket(buffer, offset)
+
+* `buffer` [&lt;Buffer>]
+* `offset` [&lt;Number>]
+* return [&lt;Object>]
+
+Reads the domain string from a packet and returns an object containing the number of bytes read and the domain.
```

Comment:
Can you please break lines at 80 char per line?

Reply:
Updated.